### DISCUSSION TITLE
Lesson10: VirtualMachineError: revert: SafeERC20: low-level call failed

### DISCUSSION THREAD

**Discussion by Cosmodude** [smartcontractkit/full-blockchain-solidity-course-py#1706] [Q&A] [answered]
I am at 9:23:25 of the course, trying to run the borrow script, and get this error:

> Depositing...
Transaction sent: 0xa46d20be116d71e0f0439f32b16bd037eda594761cb998009fe7b138eed3c010
  Gas price: 0.0 gwei   Gas limit: 6721975   Nonce: 8
  Transaction confirmed (SafeERC20: low-level call failed)   Block: 15558903   Gas used: 147385 (2.19%)
> File "D:\Python\lib\site-packages\brownie\_cli\run.py", line 51, in main
    return_value, frame = run(
  File "D:\Python\lib\site-packages\brownie\project\scripts.py", line 110, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File ".\scripts\aave_borrow.py", line 45, in main
    borrow()
  File ".\scripts\aave_borrow.py", line 22, in borrow
    tx=lend_pool.deposit(erc20_address, amount, acc.address, 0, {"from": acc} )
  File "D:\Python\lib\site-packages\brownie\network\contract.py", line 1861, in __call__
    return self.transact(*args)
  File "D:\Python\lib\site-packages\brownie\network\contract.py", line 1734, in transact
    return tx["from"].transfer(
  File "D:\Python\lib\site-packages\brownie\network\account.py", line 682, in transfer
    receipt._raise_if_reverted(exc)
  File "D:\Python\lib\site-packages\brownie\network\transaction.py", line 446, in _raise_if_reverted
    raise exc._with_attr(
VirtualMachineError: revert: SafeERC20: low-level call failed

aave_borrow.py:
`from scripts.helpful_scripts import get_account
from scripts.get_weth import get_weth
from web3 import Web3
from brownie import (
    network,
    accounts,
    config,
    interface

)
amount = Web3.toWei(0.1, "ether")
print(amount)
def borrow():
    acc=get_account()
    erc20_address= config["networks"][network.show_active()]["weth_token"]
    if network.show_active() in ["mainnet-fork"]:
        get_weth()
    lend_pool=get_lending_pool()
    print(lend_pool.address)
    approve_erc20(amount, lend_pool.address, erc20_address, acc)
    print ("Depositing...")
    tx=lend_pool.deposit(erc20_address, amount, acc.address, 0, {"from": acc} )
    tx.wait(1)
    print ("Deposited!")

#Approve sending out ERC20 Tokens
def approve_erc20(amount, spender,erc20_address, account):
    print("proving ERC20 token...")
    erc20= interface.IERC20(erc20_address)
    tx= erc20.approve(spender,amount,{"from":account})
    tx.wait(1)
    print("Approved!")
    return tx

def get_lending_pool():
    lending_pool_addresses_provider = interface.ILendingPoolAddressesProvider(
        config["networks"][network.show_active()]["lending_pool_addresses_provider"]
    )
    lending_pool_address = lending_pool_addresses_provider.getLendingPool()
    lending_pool = interface.ILendingPool(lending_pool_address)
    return lending_pool

def main():
    borrow()`

brownie-config.yaml:
`dependencies:
  - aave/protocol-v2@1.0.1
compiler:
  solc:
    remappings:
      - '@aave=aave/protocol-v2@1.0.1'

dotenv: .env

networks:
  default: development
  development:
    keyhash: "0x6c3699283bda56ad74f6b855546325b68d482e983852a7a82979cc4807b641f4"
    fee: 100000000000000000
    jobId: "29fa9aa13bf1468788b7cc4a500a45b8"
    update_interval: 60
    verify: False
  goerli: 
    weth_token: '0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6'
    lending_pool_addresses_provider: "0x5E52dEc931FFb32f609681B8438A51c675cc232d"
    eth_usd_price_feed: '0xD4a33860578De61DBAbDc8BFdb98FD742fA7028e'
    vrf_coordinator: '0x2bce784e69d2Ff36c71edcB9F88358dB0DfB55b4'
    vrf_coordinator-v2: '0x2Ca8E0C643bDe4C2E08ab1fA0da3401AdAD7734D'
    key_hash-v2: '0x79d3d8832d904592c0bf9818b621522c988bb8b0c05cdc3b15aea1b6e8db0c15'
    link_token: '0x326C977E6efc84E512bB9C30f76E30c160eD06FB'
    keyhash: '0x0476f9a745b61ea5c0ab224d3a6e4c99f0b02fce4da01143a4f70aa80ae76e8a'
    fee: 100000000000000000
    verify: True
  kovan:
    vrf_coordinator: "0xdD3782915140c8f3b190B5D67eAc6dc5760C46E9"
    link_token: "0xa36085F69e2889c224210F603D836748e7dC0088"
    keyhash: "0x6c3699283bda56ad74f6b855546325b68d482e983852a7a82979cc4807b641f4"
    fee: 100000000000000000
    oracle: "0xc57b33452b4f7bb189bb5afae9cc4aba1f7a4fd8"
    jobId: "d5270d1c311941d0b08bead21fea7747"
    eth_usd_price_feed: "0x9326BFA02ADD2366b30bacB125260Af641031331"
    # Change to True if you have an Etherscan API key and want to verify
    verify: True
    update_interval: 60
  ganache:
    keyhash: "0x6c3699283bda56ad74f6b855546325b68d482e983852a7a82979cc4807b641f4"
    fee: 100000000000000000
    jobId: "29fa9aa13bf1468788b7cc4a500a45b8"
    update_interval: 60
    verify: False
  rinkeby:
    vrf_coordinator: "0xb3dCcb4Cf7a26f6cf6B120Cf5A73875B7BBc655B"
    link_token: "0x01be23585060835e02b77ef475b0cc51aa1e0709"
    keyhash: "0x2ed0feb3e7fd2022120aa84fab1945545a9f2ffc9076fd6156fa96eaff4c1311"
    fee: 100000000000000000
    oracle: "0xc57b33452b4f7bb189bb5afae9cc4aba1f7a4fd8"
    jobId: "6b88e0402e5d415eb946e528b8e0c7ba"
    eth_usd_price_feed: "0x8A753747A1Fa494EC906cE90E9f37563A8AF630e"
    # Change to True if you have an Etherscan API key and want to verify
    verify: False
  fuji:
    link_token: "0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846"
    fee: 100000000000000000
    oracle: "0xcc80934eaf22b2c8dbf7a69e8e0d356a7cac5754"
    jobId: "5ca4fa9b2d64462290abfbda84e38cf4"
  mumbai:
    eth_usd_price_feed: "0x0715A7794a1dc8e42615F059dD6e406A6594651A"
    link_token: "0x326C977E6efc84E512bB9C30f76E30c160eD06FB"
    vrf_coordinator: "0x8C7382F9D8f56b33781fE506E897a4F1e2d17255"
    keyhash: "0x6e75b569a01ef56d18cab6a8e71e6600d6ce853834d4a5748b720d06f878b3a4"
    fee: 1000000000000000000
  binance:
    # link_token: ??
    eth_usd_price_feed: "0x9ef1B8c0E4F7dc8bF5719Ea496883DC6401d5b2e"
  binance-fork:
    eth_usd_price_feed: "0x9ef1B8c0E4F7dc8bF5719Ea496883DC6401d5b2e"
  mainnet-fork:
    weth_token: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
    lending_pool_addresses_provider: "0xB53C1a33016B2DC2fF3653530bfF1848a515c8c5"
    eth_usd_price_feed: "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419"
  matic-fork:
    eth_usd_price_feed: "0xF9680D99D6C9589e2a93a78A04A279e509205945"
wallets:
  from_key: ${PRIVATE_KEY}`

I will be grateful if you help me to solve the problem, since I am stuck.

**Comment by cromewar** [comment, marked as answer]
Hello @Cosmodude 
Due to the recent merge and goerli being the only available test network, as is recommended on #1690 and explained on #1703, please skip the lesson 10 for now.
Thanks.

**Reply by Cosmodude** [reply]
Ok, thank you!
